submissions/Oleksii_Drohachov/Frogger_task/app.js

```diff
@@ -0,0 +1,157 @@
+const entity = {
+  width: 80,
+  height: 60,
+};
+
+const Enemy = function (yAxisPosition, moveSpeed, width, height) {
+  this.xAxisPosition = -80;
+  this.yAxisPosition = yAxisPosition;
+  this.moveSpeed = moveSpeed;
+  this.width = width;
+  this.height = height;
+  this.sprite = "images/enemy-bug.png";
+};
+
+Enemy.prototype.update = function (dt) {
+  this.xAxisPosition += this.moveSpeed * dt;
+
+  if (this.xAxisPosition >= ctx.canvas.width) {
+    this.xAxisPosition = -80;
+  }
+
+  this.checkCollision();
+};
+
+Enemy.prototype.render = function () {
+  ctx.drawImage(
+    Resources.get(this.sprite),
+    this.xAxisPosition,
+    this.yAxisPosition
+  );
+};
+
+Enemy.prototype.checkCollision = function () {
+  if (
+    player.xAxisPosition < this.xAxisPosition + this.width &&
+    player.width + player.xAxisPosition > this.xAxisPosition &&
+    player.yAxisPosition < this.yAxisPosition + this.height &&
+    player.yAxisPosition + player.height > this.yAxisPosition
+  ) {
+    player.score = 0;
+    scoreTitle.textContent = `Current score is: ${player.score}`;
+    player.respawn();
+  }
+};
+
+const enemyStats = [
+  {
+    yAxisPosition: 60,
+    moveSpeed: 180,
+  },
+  {
+    yAxisPosition: 140,
+    moveSpeed: 140,
+  },
+  {
+    yAxisPosition: 220,
+    moveSpeed: 100,
```

Comment:
As a peer developer I am tasked to add 5 more enemies.
I want to do it with the least effort possible. The numbers here look mysteriously.
What I see and know is that the play field has clear rows. Ideally, I'd like to define a row for an enemy and let the program calculate pixels.